lib/index.ts

```diff
@@ -0,0 +1,6 @@
+export { logger } from './logger';
```

Comment:
@rarkins Do you need something else to be exported?

Reply:
I do, but maybe I won't need them long term and we can skip them for now. In the app I use util/http/github, workers/global/config/parse, util/host-rules, config, workers/repository, and util/cache

Reply:
Ok, but deep imports are no longer possible with esm, only explicit defined exports are usable.

Reply:
But they can be added as-needed later, right?

Reply:
Yes

Reply:
I've added what's needed for our internal tools